### ISSUE TITLE
MACs calculation for ConvTranspose

### ISSUE THREAD

**Issue by zgerem** [ThanatosShinji/onnx-tool#32]
I have a model with ConvTranspose, and I am profiling it with onnx_tool.model_profile. the problem is that the MACs is not calculated correctly for this layer type. I expect the calculated MACs to be: input_height*input_width*num_in_channels*num_out_channels*kernel_height*kernel_width + out_height*out_width*num_out_channels. 

could you check that?

**Comment by ThanatosShinji** [comment]
What's the attributes of this transconv?strides?

**Comment by ThanatosShinji** [comment]
> input_height x input_width x num_in_channels x num_out_channels x kernel_height x kernel_width + out_height x out_width x num_out_channels

this is not correct for TransposeConv's MACs. Are you trying to escape the padding area of the src tensor from computation?
